docs/website/views/_includes/navigation/sidebar-item.njk

```diff
@@ -6,7 +6,7 @@
     <esl-trigger class="sidebar-nav-item-trigger sidebar-nav-item-arrow "
                  target="::parent::next"
                  {% if isPrimaryActive %}active{% endif %}>
-      {{ item.data.title }}
+      {{ item.data.navTitle or item.data.title }}
```

Comment:
mb we can leave `item.data.title` here, because it's not our article pages, it's just a trigger
<img width="268" alt="Screenshot 2022-10-19 at 16 33 17" src="https://user-images.githubusercontent.com/52083788/196721343-105e7ebc-0030-4cb0-b152-12cbb40fd737.png">

Reply:
Yep, That's true.
But I propose making it consistent instead for future
@abarmina, @NastaLeo , what do you think?

Reply:
I do not care, it's not a critical moment